lib/primitives.js

```diff
@@ -1,218 +1,258 @@
-var _ = require('lodash');
+var ArrayType, BoolType, ElementType, KeyValueType, NullType, NumberType, ObjectType, StringType, convertFromDom, convertToType, _,
```

Comment:
I have not followed this, but should before merging. 

http://stackoverflow.com/a/13646337/4175777